paper/myst.yml

```diff
@@ -2,48 +2,56 @@
 version: 1
 project:
   id: ubcgif-2020-heagy-etal-exploration-geophysics
-  # title: TITLE OF PAPER
-  # description: SHORT DESCRIPTION HERE
+  title: Open-source software for simulations and inversions of airborne electromagnetic data
+  description: overview of the software in its application to airborne electromagnetics and demonstrate its use for visualising fields and fluxes in a forward simulation, as well as its flexibility in formulating and solving the inverse problem.
   authors:
-    # - name: AUTHOR 1
-    #   email: EMAIL
-    #   corresponding: true
-    #   orcid: ORCID
-    #   affiliation:
-    #     id: ubc
-    #     name: Department of Earth, Ocean and Atmospheric Sciences, University of British Columbia, Vancouver, British Columbia
-    # - name: Lindsey J. Heagy
-    #   email: lheagy@eoas.ubc.ca
-    #   corresponding: true
-    #   orcid: 0000-0002-1551-5926
-    #   affiliation: ubc
-  # doi: DOI
-  # arxiv: ARXIV
-  # github: GITHUB-URL
-  # date: YYYY/MM/DD # date of publication
+    - name: Lindsey J. Heagy
+      email: lheagy@eoas.ubc.ca
+      corresponding: true
+      orcid: 0000-0002-1551-5926
+      affiliation:
+        id: ubc
+        name: Department of Earth, Ocean and Atmospheric Sciences, University of British Columbia, Vancouver, British Columbia
+    - name: Seogi Kang
+      email: sgkang09@stanford.edu
+      orcid: 0000-0002-9963-936X
+      affiliation: ubc
+    - name: Rowan Cockett
+      email: rowan@curvenote.com 
+      orcid: 0000-0002-7859-8394
+      affiliation: ubc
+    - name: Douglas W. Oldenburg
+      email: doug@eos.ubc.ca
+      orcid: 0000-0002-4327-2124
+      affiliation: ubc
+  doi: 10.1080/08123985.2019.1583538
+  arxiv: 1902.08238
+  github: https://github.com/simpeg-research/heagy-2018-AEM
+  date: 2019/06/10 # date of publication
```

Comment:
Lets update this to 2020, since that is then consistent with the publication year (if we need a month and day, then 2020/01/01 works